projects/demo/src/modules/directives/title/examples/1/index.ts

```diff
@@ -1,12 +1,14 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiTitleDirective} from '@taiga-ui/core';
 
 @Component({
-    selector: 'tui-header-example-1',
+    standalone: true,
+    imports: [TuiTitleDirective],
     templateUrl: './index.html',
     styleUrls: ['./index.less'],
     encapsulation,
     changeDetection,
 })
-export class TuiHeaderExample1 {}
+export default class ExampleComponent {}
```

Comment:
Do we need the name for the class?

Reply:
yes, we need, otherwise the error `ERROR Error: The component 'default_1' needs to be compiled using the JIT compiler` will appear